tests/Api/Responses/Expected/shop/product/get_product_attributes_collection_response.json

```diff
@@ -0,0 +1,99 @@
+{
+    "@context": "\/api\/v2\/contexts\/ProductAttributeValue",
+    "@id": "\/api\/v2\/shop\/products\/MUG_SW\/attributes",
+    "@type": "hydra:Collection",
+    "hydra:member": [
+        {
+            "@id": "\/api\/v2\/shop\/product-attribute-values\/@integer@",
+            "@type": "ProductAttributeValue",
+            "id": @integer@,
+            "attribute": "\/api\/v2\/shop\/product-attributes\/dishwasher_safe",
+            "localeCode": "pl_PL",
+            "name": "mozna myc w zmywarce",
+            "type": "checkbox",
+            "code": "dishwasher_safe",
+            "value": false
+        },
+        {
+            "@id": "\/api\/v2\/shop\/product-attribute-values\/@integer@",
+            "@type": "ProductAttributeValue",
+            "id": @integer@,
+            "attribute": "\/api\/v2\/shop\/product-attributes\/manufactured",
```

Comment:
I'm strongly considering embedding attributes here, to ease the usage of this feature. WDYT?

Reply:
I don't think there is some extra info in `shop` that makes attributes necessary to embed but in `admin` I think it'll be a good idea. Attribute's properties like `name`, `code`, and `type` are accessible through AttributeValue's getters: https://github.com/Sylius/Sylius/blob/ee9116e81e9ecf2130cf0609f5471a89a7244ea8/src/Sylius/Component/Attribute/Model/AttributeValue.php#L106L125